code/modules/day_night/day_night_controller.dm

```diff
@@ -0,0 +1,251 @@
+/**
+ * Day Night Presets are easy to use day/night cycles for certain maps, they are what is used to dictate the sky color.
+ *
+ * IMPORTANT: The lightzones in this MUST all add up to 23, you cannot miss an hour!
```

Comment:
what happens if you don't?

This could be a unit test if it's going to cause bugs

Reply:
In testing, not having all of the hours filled out will cause the cycle to runtime as there would be no values return within the specified timezone, resulting in blackness.

Reply:
unit test this then

Reply:
done